2023-10-16/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import "fmt"
+
+func concatenate(a, b string) string {
+	return a + b
+}
+
+func swap(a, b string) (string, string) {
+	return b, a
```

Comment:
Actually trickier than it looks, or maybe less tricky than it looks im unsure 🤓
But here we define a new function called swap. It contains all the same parameters as the same concatenate function we defined earlier. The function itself return two strings but the variables are stated in the opposite order